test/validation/date.validation.spec.ts

```diff
@@ -1,10 +1,12 @@
 const mockIf = jest.fn();
+const mockTrim = jest.fn();
 const mockCustom = jest.fn();
 const mockEquals = jest.fn();
 const mockNotEmpty = jest.fn();
 
 jest.mock('express-validator', () => ({
   body: jest.fn().mockImplementation(() => ({
+    trim: mockTrim.mockReturnThis(),
```

Comment:
there should be a test in here somewhere where you can add expect(mockTrim).toHaveBeenCalled
maybe line 94